LICENSE-MIT

```diff
@@ -1,5 +1,3 @@
-Copyright (c) 2010 The Rust Project Developers
```

Comment:
I'm having trouble squaring this removal with the “[The above copyright notice and this permission notice
shall be included…][1]” sentence below.  @joshtriplett, do you think that will be a problem, now that there is no longer an above copyright notice?

[1]: https://github.com/rust-lang/rust/pull/43498/commits/2a8807e889a43c6b89eb6f2736907afa87ae592f#diff-7f0bdbc0a0545831259b66259ac6b604R11

Reply:
I don't believe so, no. The requirement is to reproduce what's there; if something isn't there you don't need to reproduce it.